frontend/src/components/ContributionMessages/ContributionMessagesFormular.spec.js

```diff
@@ -0,0 +1,93 @@
+import { mount } from '@vue/test-utils'
+import ContributionMessagesFormular from './ContributionMessagesFormular.vue'
+import { toastErrorSpy, toastSuccessSpy } from '../../../test/testSetup'
+
+const localVue = global.localVue
+
+const apolloMutateMock = jest.fn().mockResolvedValue()
+
+describe('ContributionMessagesFormular', () => {
+  let wrapper
+
+  const propsData = {
+    contributionId: 42,
+  }
+
+  const mocks = {
+    $t: jest.fn((t) => t),
+    $apollo: {
+      mutate: apolloMutateMock,
+    },
+    $i18n: {
+      locale: 'en',
+    },
+  }
+
+  const Wrapper = () => {
+    return mount(ContributionMessagesFormular, {
+      localVue,
+      mocks,
+      propsData,
+    })
+  }
+
+  describe('mount', () => {
+    beforeEach(() => {
+      wrapper = Wrapper()
+    })
+
+    it('has a DIV .contribution-messages-formular', () => {
+      expect(wrapper.find('div.contribution-messages-formular').exists()).toBe(true)
+    })
+
+    describe('call onReset', () => {
```

Comment:
I would more prefer to `on trigger reset` here like:
https://github.com/gradido/gradido/blob/0da776b2c6eae30f4e790cbc04884b365b6969b7/frontend/src/components/Contributions/ContributionForm.spec.js#L291-310

Reply:
ok